static/styles/news/news.scss

```diff
@@ -33,3 +33,13 @@
 .tabContainer {
   clear: both;
 }
+.icon-custom {
+  vertical-align: middle;
+}
+.custom-style {
+  font-size: 100% !important;
+}
+.tooltip-inner {
+  max-width: 100%;
+  text-align: left !important;
+}
```

Comment:
Are you sure we don't have classes with the same name? It seems too global to me. Please choose explicit namespace news-*